hack/.golint_failures

```diff
@@ -423,6 +423,7 @@ staging/src/k8s.io/apimachinery/pkg/watch
 staging/src/k8s.io/apiserver/pkg/admission
 staging/src/k8s.io/apiserver/pkg/admission/configuration
 staging/src/k8s.io/apiserver/pkg/admission/initializer
+staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook
```

Comment:
can we fix the warnings?

Reply:
the warning is from rebasing to splitting webhook type
```
staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors.go|25 col 6 warning| type name will be used as webhook.WebhookAccessor
```

Reply:
This was done in the Webhook API type split PR. The warning we silenced is that `webhook.WebhookAccessor` stutters, @liggitt and I were in favor of keeping the `WebhookAccessor` name instead of renaming it to just `Accessor` or something. Happy to revisit this.

Reply:
just call it Accessor. But that's something for @jpbetz as a followup, not htis PR.